libs/components/lookup/testing/src/autocomplete/autocomplete-input-harness.ts

```diff
@@ -63,9 +63,9 @@ export class SkyAutocompleteInputHarness extends ComponentHarness {
   }
 
   /**
-   * Returns the value of the input's `aria-owns` attribute.
+   * Returns the value of the input's `aria-controls` attribute.
    */
   public async getAriaOwns(): Promise<string | null> {
-    return (await this.host()).getAttribute('aria-owns');
+    return (await this.host()).getAttribute('aria-controls');
```

Comment:
The name of this method conflicts with the type of value returned. Why did we have to use `aria-controls` instead of `aria-owns`?

Reply:
There was a [change in axe-core 4.6](https://github.com/dequelabs/axe-core/pull/3786) that made `aria-controls` required on `role="combobox"` elements and changes in 4.7 clarified rules around `combobox` and referenced elements.

Without switching from `aria-owns` to `aria-controls`, this is the error:

```
[Rule: 'aria-required-attr'] Required ARIA attributes must be provided - WCAG: wcag412
 Get help at: https://dequeuniversity.com/rules/axe/4.7/aria-required-attr?application=axeAPI

[CRITICAL] Fix any of the following:
  - Required ARIA attribute not present: aria-controls
Target: #sky-autocomplete-wrapper-91
HTML: <div _ngcontent-a-c80="" aria-autocomplete="list" aria-haspopup="listbox" role="combobox" class="sky-autocomplete" aria-expanded="true" aria-labelledby="my-autocomplete-label" id="sky-autocomplete-wrapper-91" aria-owns="sky-autocomplete-list-91">
  - [aria-required-attr] Required ARIA attribute not present: aria-controls
```

Reply:
Good call.